Index: swing-application-impl/src/main/java/org/cytoscape/internal/CyActivator.java
===================================================================
--- swing-application-impl/src/main/java/org/cytoscape/internal/CyActivator.java	(revision 28196)
+++ swing-application-impl/src/main/java/org/cytoscape/internal/CyActivator.java	(working copy)
@@ -171,10 +171,10 @@
 		SubmenuTaskManager submenuTaskManagerServiceRef = getService(bc, SubmenuTaskManager.class);
 		RenderingEngineFactory dingRenderingEngineFactoryServiceRef = getService(bc,
 		                                                                         RenderingEngineFactory.class,
-		                                                                         "(id=ding)");
+		                                                                         "(id=wind)");
 		RenderingEngineFactory dingNavigationPresentationFactoryServiceRef = getService(bc,
 		                                                                                RenderingEngineFactory.class,
-		                                                                                "(id=dingNavigation)");
+		                                                                                "(id=windMap)");
 		CyProperty bookmarkServiceRef = getService(bc, CyProperty.class,
 		                                           "(cyPropertyName=bookmarks)");
 		BookmarksUtil bookmarksUtilServiceRef = getService(bc, BookmarksUtil.class);
Index: swing-application-impl/src/main/java/org/cytoscape/internal/view/NetworkViewManager.java
===================================================================
--- swing-application-impl/src/main/java/org/cytoscape/internal/view/NetworkViewManager.java	(revision 28196)
+++ swing-application-impl/src/main/java/org/cytoscape/internal/view/NetworkViewManager.java	(working copy)
@@ -112,7 +112,7 @@
 
 	// TODO: for now, use this as default. But in the future, we should provide
 	// UI to select presentation.
-	private static final String DEFAULT_PRESENTATION = "ding";
+	private static final String DEFAULT_PRESENTATION = "wind";
 
 	private final CyNetworkViewManager networkViewManager;
 	private final CyApplicationManager applicationManager;
