Index: vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/DefaultViewEditorImpl.java
===================================================================
--- vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/DefaultViewEditorImpl.java	(revision 28196)
+++ vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/DefaultViewEditorImpl.java	(working copy)
@@ -201,7 +201,7 @@
 
 		for (VisualLexicon lexicon : lexSet) {
 			for (VisualProperty<?> vp : props) {
-				if (lexicon.getVisualLexiconNode(vp).getChildren().size() == 0)
+				if (lexicon.getVisualLexiconNode(vp) != null && (lexicon.getVisualLexiconNode(vp).getChildren().size() == 0))
 					propSet.add(vp);
 			}
 		}
@@ -218,7 +218,7 @@
 
 		for (VisualLexicon lexicon : lexSet) {
 			for (VisualProperty<?> vp : props) {
-				if (lexicon.getVisualLexiconNode(vp).getChildren().size() == 0
+				if (lexicon.getVisualLexiconNode(vp) != null && lexicon.getVisualLexiconNode(vp).getChildren().size() == 0
 						&& lexicon.getVisualLexiconNode(vp).getParent()
 								.getVisualProperty() == MinimalVisualLexicon.NETWORK)
 					propSet.add(vp);
Index: vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/bypass/BypassManager.java
===================================================================
--- vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/bypass/BypassManager.java	(revision 28196)
+++ vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/bypass/BypassManager.java	(working copy)
@@ -40,7 +40,7 @@
 	}
 
 	public void addBypass(RenderingEngineFactory<?> factory, Map props) {
-		if (props.containsValue("ding") == false)
+		if (props.containsValue("wind") == false)
 			return;
 
 		final VisualLexicon lexicon = factory.getVisualLexicon();
Index: vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/DefaultViewPanelImpl.java
===================================================================
--- vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/DefaultViewPanelImpl.java	(revision 28196)
+++ vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/DefaultViewPanelImpl.java	(working copy)
@@ -57,6 +57,7 @@
 import org.cytoscape.view.presentation.RenderingEngine;
 import org.cytoscape.view.presentation.RenderingEngineFactory;
 import org.cytoscape.view.presentation.property.MinimalVisualLexicon;
+import org.cytoscape.view.presentation.property.RichVisualLexicon;
 import org.cytoscape.view.vizmap.VisualStyle;
 import org.cytoscape.view.vizmap.gui.DefaultViewPanel;
 import org.cytoscape.view.vizmap.gui.SelectedVisualStyleManager;
@@ -124,6 +125,10 @@
 		final CyNode source = dummyNet.addNode();
 		final CyNode target = dummyNet.addNode();
 
+		// Add dummy nodes to balance the z coordinates of the other nodes
+		final CyNode balance1 = dummyNet.addNode();
+		final CyNode balance2 = dummyNet.addNode();
+		
 		dummyNet.getRow(source).set(CyTableEntry.NAME, "Source");
 		dummyNet.getRow(target).set(CyTableEntry.NAME, "Target");
 
@@ -135,11 +140,23 @@
 				.createNetworkView(dummyNet);
 
 		// Set node locations
-		dummyview.getNodeView(source).setVisualProperty(NODE_X_LOCATION, 0d);
-		dummyview.getNodeView(source).setVisualProperty(NODE_Y_LOCATION, 0d);
-		dummyview.getNodeView(target).setVisualProperty(NODE_X_LOCATION, 150d);
-		dummyview.getNodeView(target).setVisualProperty(NODE_Y_LOCATION, 20d);
-
+		dummyview.getNodeView(source).setVisualProperty(NODE_X_LOCATION, -60.0);
+		dummyview.getNodeView(source).setVisualProperty(NODE_Y_LOCATION, -50.0);
+		dummyview.getNodeView(source).setVisualProperty(RichVisualLexicon.NODE_Z_LOCATION, 0.0);
+		
+		dummyview.getNodeView(target).setVisualProperty(NODE_X_LOCATION, 60.0);
+		dummyview.getNodeView(target).setVisualProperty(NODE_Y_LOCATION, 50.0);
+		dummyview.getNodeView(target).setVisualProperty(RichVisualLexicon.NODE_Z_LOCATION, 0.0);
+		
+		dummyview.getNodeView(balance1).setVisualProperty(NODE_X_LOCATION, -500.0);
+		dummyview.getNodeView(balance1).setVisualProperty(NODE_Y_LOCATION, -500.0);
+		dummyview.getNodeView(balance1).setVisualProperty(RichVisualLexicon.NODE_Z_LOCATION, -450.0);
+		
+		dummyview.getNodeView(balance2).setVisualProperty(NODE_X_LOCATION, 500.0);
+		dummyview.getNodeView(balance2).setVisualProperty(NODE_Y_LOCATION, 500.0);
+		dummyview.getNodeView(balance2).setVisualProperty(RichVisualLexicon.NODE_Z_LOCATION, -450.0);
+		
+		
 		final VisualStyle currentStyle = selectedManager.getCurrentVisualStyle();
 		currentStyle.apply(dummyview);
 
Index: vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/CyActivator.java
===================================================================
--- vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/CyActivator.java	(revision 28196)
+++ vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/CyActivator.java	(working copy)
@@ -74,8 +74,8 @@
 		CyApplicationConfiguration cyApplicationConfigurationServiceRef = getService(bc,CyApplicationConfiguration.class);
 		DialogTaskManager dialogTaskManagerServiceRef = getService(bc,DialogTaskManager.class);
 		VisualStyleFactory visualStyleFactoryServiceRef = getService(bc,VisualStyleFactory.class);
-		VisualLexicon dingVisualLexiconServiceRef = getService(bc,VisualLexicon.class,"(id=ding)");
-		RenderingEngineFactory dingRenderingEngineFactoryServiceRef = getService(bc,RenderingEngineFactory.class,"(id=ding)");
+		VisualLexicon windVisualLexiconServiceRef = getService(bc,VisualLexicon.class,"(id=wind)");
+		RenderingEngineFactory windRenderingEngineFactoryServiceRef = getService(bc,RenderingEngineFactory.class,"(id=wind)");
 		VisualMappingManager vmmServiceRef = getService(bc,VisualMappingManager.class);
 		CyNetworkManager cyNetworkManagerServiceRef = getService(bc,CyNetworkManager.class);
 		CyApplicationManager cyApplicationManagerServiceRef = getService(bc,CyApplicationManager.class);
Index: vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/util/VisualPropertyFilter.java
===================================================================
--- vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/util/VisualPropertyFilter.java	(revision 28196)
+++ vizmap-gui-impl/src/main/java/org/cytoscape/view/vizmap/gui/internal/util/VisualPropertyFilter.java	(working copy)
@@ -12,8 +12,8 @@
 
 	// Visual Properties which are not compatible with current rendering engine.
 	static {
-		INCOMPATIBLE_VP_SET.add(RichVisualLexicon.NODE_DEPTH);
-		INCOMPATIBLE_VP_SET.add(RichVisualLexicon.NODE_Z_LOCATION);
+		// INCOMPATIBLE_VP_SET.add(RichVisualLexicon.NODE_DEPTH);
+		// INCOMPATIBLE_VP_SET.add(RichVisualLexicon.NODE_Z_LOCATION);
 	}
 
 	
Index: vizmap-gui-impl/src/main/resources/default_vizmap.xml
===================================================================
--- vizmap-gui-impl/src/main/resources/default_vizmap.xml	(revision 28196)
+++ vizmap-gui-impl/src/main/resources/default_vizmap.xml	(working copy)
@@ -458,7 +458,7 @@
             <visualProperty name="NODE_TRANSPARENCY" default="255"/>
             <visualProperty name="NODE_SIZE" default="35.0"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_2"/>
-            <visualProperty name="NODE_FILL_COLOR" default="#00acad"/>
+            <visualProperty name="NODE_FILL_COLOR" default="#797979"/>
             <visualProperty name="NODE_CUSTOMPAINT_5" default="org.cytoscape.view.presentation.property.DefaultVisualizableVisualProperty$VisualizableImpl@69d41410"/>
             <visualProperty name="NODE_CUSTOMPAINT_6" default="org.cytoscape.view.presentation.property.DefaultVisualizableVisualProperty$VisualizableImpl@69d41410"/>
             <visualProperty name="NODE_CUSTOMPAINT_7" default="org.cytoscape.view.presentation.property.DefaultVisualizableVisualProperty$VisualizableImpl@69d41410"/>
@@ -470,7 +470,7 @@
             <visualProperty name="NODE_LABEL_FONT_FACE" default="Dialog,plain,12"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_SIZE_7" default="50.0"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_8"/>
-            <visualProperty name="NODE_DEPTH" default="0.0"/>
+            <visualProperty name="NODE_DEPTH" default="70.0"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_SIZE_9" default="50.0"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_5"/>
             <visualProperty name="NODE_SELECTED" default="false"/>
@@ -495,7 +495,7 @@
             <visualProperty name="NODE_TOOLTIP" default=""/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_POSITION_9" default="C,C,c,0.00,0.00"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_SIZE_6" default="50.0"/>
-            <visualProperty name="NODE_HEIGHT" default="40.0"/>
+            <visualProperty name="NODE_HEIGHT" default="70.0"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_7"/>
             <visualProperty name="NODE_CUSTOMPAINT_2" default="org.cytoscape.view.presentation.property.DefaultVisualizableVisualProperty$VisualizableImpl@69d41410"/>
             <visualProperty name="NODE_CUSTOMGRAPHICS_POSITION_6" default="C,C,c,0.00,0.00"/>
